code/modules/research/xenoarchaeology/artifact/effects/unknown_effect_reagentblock.dm

```diff
@@ -0,0 +1,42 @@
+/datum/artifact_effect/reagentblock
+	effecttype = "reagentblock"
+	var/duration = 0
+	copy_for_battery = list("duration")
+
+/datum/artifact_effect/reagentblock/New()
+	..()
+	effect_type = pick(5,7)
+	duration = rand(600,6000)
+	if(effect == EFFECT_AURA)
+		effectrange = rand(1,5)
+
+/datum/artifact_effect/reagentblock/DoEffectTouch(var/mob/living/user)
+	var/weakness = GetAnomalySusceptibility(user)
+	if(ishuman(user) && prob(weakness * 100))
```

Comment:
Because things that aren't humans don't really do much with reagents. Monkeys and xenos work I guess, but why bother